notebooks/logging-examples/logging-experiment-failures.ipynb

```diff
@@ -0,0 +1,157 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "3b466d92",
+   "metadata": {},
+   "source": [
+    "## Distinguishing Failed Experiments with Rubicon \n",
+    "Modeling runs can fail for a number of reasons. When logging with rubicon_ml, a failed run may result in an empty or incomplete experiment. In this example, we'll walk through how to handle such experiments."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d4f58990",
+   "metadata": {},
+   "source": [
+    "First lets simulate the problem. To do this we'll create an estimator that will fail on it's `fit()` 30% of the time.  We will consider any experiment that has a `none` value for `self.state`."
```

Comment:
last sentence isn't quite finished

_We will consider any pipeline that has a learned attribute `self.state_` to have "succeeded," and any that does not to have "failed."_

the experiments' success/failure depends on the attributes of the pipeline, not the experiment itself. the experiment is just tracking that info